aio.core/aio/core/functional/utils.py

```diff
@@ -9,10 +9,15 @@
 
 from trycast import trycast  # type:ignore
 
-import orjson as json
```

Comment:
also - small nit can we keep the import orders - i generally put external libs before any locally maintained ones